04_containers.Rmd

```diff
@@ -347,7 +347,25 @@ library(curatedMetagenomicData)
 tse <- curatedMetagenomicData("Vatanen*", dryrun = FALSE, counts = TRUE)
 ```
 
+###  Human microbiome compendium
 
+[MicroBioMap](https://seandavi.github.io/MicroBioMap/) dataset includes 
+over 170k samples of publicly available 16S rRNA amplicon sequencing data, 
+all processed using the same pipeline and reference database[@Abdill2023].
+
+Load the compendium using:
```

Comment:
"After installing the MicroBioMap package (see the original website for instructions), you can load the compendium with:"